examples/flutter_gallery/lib/demo/shrine/shrine_order.dart

```diff
@@ -0,0 +1,227 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/material.dart';
+
+import '../shrine_demo.dart' show ShrinePageRoute;
+import 'shrine_page.dart';
+import 'shrine_theme.dart';
+import 'shrine_types.dart';
+
+/// Describes a product and vendor in detail, supports specifying
+/// a order quantity (0-5). Appears at the top of the OrderPage.
+class OrderItem extends StatelessWidget {
+  OrderItem({ Key key, this.product, this.quantity, this.quantityChanged }) : super(key: key) {
+    assert(product != null);
+    assert(quantity != null && quantity >= 0 && quantity <= 5);
+  }
+
+  final Product product;
+  final int quantity;
+  final ValueChanged<int> quantityChanged;
+
+  @override
+  Widget build(BuildContext context) {
+    final ShrineTheme theme = ShrineTheme.of(context);
+    return new Material(
+      type: MaterialType.card,
+      elevation: 0,
+      child: new Padding(
+        padding: const EdgeInsets.only(left: 16.0, top: 18.0, right: 16.0, bottom: 24.0),
+        child: new Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            new Padding(
+              padding: const EdgeInsets.only(left: 56.0),
+              child: new SizedBox(
+                width: 248.0,
+                height: 248.0,
+                child: new Hero(
+                  tag: productHeroTag,
+                  child: new NetworkImage(
+                    fit: ImageFit.contain,
+                    src: product.imageUrl
+                  )
+                )
+              )
+            ),
+            new SizedBox(height: 24.0),
+            new Row(
+              children: <Widget>[
+                new Padding(
+                  padding: const EdgeInsets.symmetric(horizontal: 16.0),
+                  child: new Center(
+                    child: new Icon(
+                      icon: Icons.info_outline,
+                      size: 24.0,
+                      color: const Color(0xFFFFE0E0)
+                    )
+                  )
+                ),
+                new Flexible(
+                  child: new Text(product.name, style: theme.featureTitleStyle)
+                )
+              ]
+            ),
+            new Padding(
+              padding: const EdgeInsets.only(left: 56.0),
+              child: new Column(
+                crossAxisAlignment: CrossAxisAlignment.stretch,
+                children: <Widget>[
+                  new SizedBox(height: 24.0),
```

Comment:
is this the same as: 
  Padding:  top=24, bottom=16
    Text: product.description
?